share/completions/pactl.fish

```diff
@@ -7,7 +7,9 @@
 
 
 # HACK: This is the list of commands from pacmd - used so we can use complete -w there
-set -l commands (pacmd help | string match -r '^ +[-\w]+' | string trim)
+if command -sq pacmd
+    set commands (pacmd help | string match -r '^ +[-\w]+' | string trim)
+end
```

Comment:
Soooo.... if this isn't used, this should at least be setting $commands to $ctlcommands, or the subcommand detection should fail

Reply:
OK I just did that but realised there are other completions such as `load-module` that are broken too.

Reply:
Am I understanding correclty that really it's *just* `load-module`, because that checks `pulseaudio --dump-modules`, which won't be available in this system.

Not that I think loading modules for pulseaudio makes sense anyway because you're not running pulseaudio?

Reply:
It just happened to be the first random command I tested. I haven't tested anything else or looked inside all the other pulseaudio helper functions yet so there may be others.

Load-modules probably doesn't do anything when not running native pulseaudio, but fish should at least avoid spamming the terminal with errors if it can.

Reply:
Oh, sure, it shouldn't spam. But I'm not seeing any other commands failing, as they just call `pactl`.

Reply:
Just checked and that is indeed true.